opening_hours.js

```diff
@@ -70,6 +70,23 @@
                 "Kerstmis - Noël"                                          : [ 12, 25 ]
             }
         }, /* }}} */
+        'pl': { /* {{{ */
+            'PH': { // https://pl.wikipedia.org/wiki/Dni_wolne_od_pracy_w_Polsce
+                "Nowy Rok"                                      : [ 1, 1 ],
+                "Święto Trzech Króli"                           : [ 1, 6 ],
+                "Wielkanoc"                                     : [ 'easter', 0 ],
+                "Lany Poniedziałek - drugi dzień Wielkiej Nocy" : [ 'easter', 1 ],
+                "Pierwszy Maja"                                 : [ 5, 1 ],
+                "Trzeci Maja"                                   : [ 5, 3 ],
+                "Zielone Świątki"                               : [ 'easter', 49 ],
+                "Boże Ciało"                                    : [ 'easter', 60 ],
+		"Wniebowzięcie Najświętszej Maryi Panny"        : [ 8, 15 ],
+                "Wszystkich Świętych"                           : [ 11, 1 ],
+                "Święto Niepodległości"                         : [ 11, 11 ],
+                "pierwszy dzień Bożego Narodzenia"              : [ 12, 25 ]
```

Comment:
13 kwi 2016 19:34 "Robin Schneider" <notifications@github.com> napisał(a):

> You missed the ,. That suggests that you did not run the test framework
Merging it anyway.

That's because I'm new to the GitHub and I've only just worked out all the
pull request thing ;)